sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -714,6 +714,7 @@ def eventually_apply() -> Tuple[None, bool]:
     assert all(v is None for v in online_features["value"])
 
 
+@pytest.mark.skip
```

Comment:
some issue with rebase probably? shouldn't be skipped anymore

Reply:
Fixed.